src/libraries/System.Runtime.InteropServices.RuntimeInformation/src/System/Runtime/InteropServices/RuntimeInformation/RuntimeInformation.Browser.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Runtime.InteropServices
+{
+    public static partial class RuntimeInformation
+    {
+        public static bool IsOSPlatform(OSPlatform osPlatform) => osPlatform.Equals(OSPlatform.Browser);
+
+        public static string OSDescription => "web";
+
+        public static Architecture OSArchitecture => Architecture.Wasm;
+
+        public static Architecture ProcessArchitecture => throw new PlatformNotSupportedException();
+    }
+}
```

Comment:
Are there tests we should be adding?  I'm not sure where we are on being able to run them.

Reply:
We are working on getting the tests running but still not there but this functionality is blocking the filtering. I'd prefer to fix the tests once we have them running instead of doing blind test fixes.

Reply:
How/where are we tracking the places we're introducing test holes?

Reply:
We are not tracking it yet as we have not done the initial pass